ssl/statem/statem_clnt.c

```diff
@@ -131,6 +131,10 @@ static int ossl_statem_client13_read_transition(SSL *s, int mt)
         break;
 
     case TLS_ST_CR_CERT_REQ:
+        if (s->post_handshake_auth == SSL_PHA_REQUESTED) {
```

Comment:
Yes, I see that. After receiving a CertRequest in post-handshake auth, there should be no further messages from the server, and the client should just send it's Certificate/CertVerify/Finished sequence. But, I'm not seeing it being hit in my test, so it appears that it can be removed.

Reply:
Yes, it shouldn't ever get here. This point in the code is about working out what state to move to if we read another message after the cert req. But after a cert req message is received it goes immediately into the writing state so we never attempt to read another message. This looks like dead code.

Reply:
Agreed.